src/main/groovy/generator/AppGenerator.groovy

```diff
@@ -148,7 +152,7 @@ public class AppGenerator {
 
 		println "\tGenerating dynamic code"
 
-		classes.each({ it.generateMethods(classes, true, true, true, true)})
+		classes.each({ it.generateMethods(classes, Config.shouldLogicCode, true, true, true)})
```

Comment:
Don't pass it to `generateMethods`. But use it from `generateMethods` like we do for all other `Config.SOMETHIG`